salt/modules/virtualenv_mod.py

```diff
@@ -360,4 +345,33 @@ def get_resource_content(venv, package_or_requirement, resource_name):
     if not os.path.exists(bin_path):
         raise salt.exceptions.CommandExecutionError("Path does not appear to be a virtualenv: '{0}'".format(bin_path))
 
-    return __salt__['cmd.exec_code'](bin_path, "import pkg_resources; print pkg_resources.resource_string('{0}', '{1}')".format(package_or_requirement, resource_name))
+    ret = __salt__['cmd.exec_code_all'](bin_path, "import pkg_resources; print pkg_resources.resource_string('{0}', '{1}')".format(package_or_requirement, resource_name))
+
+    if ret['retcode'] != 0:
```

Comment:
Well, the return code zero of the OS means "success" or "I don't need to look at that"